test/testcafe/spec/EnrollProfileUpdateView_spec.js

```diff
@@ -47,31 +48,33 @@ test.requestHooks(xhrEnrollProfileUpdateMock)('should have correct form title, f
   const identityPage = await setup(t);
   await identityPage.fillIdentifierField('test');
   await identityPage.fillPasswordField('test 123');
-  await identityPage.clickNextButton();
+  await identityPage.clickSignInButton();
 
   await t.expect(enrollProfileUpdatePage.getFormTitle()).eql('Additional Profile information');
-  await t.expect(await enrollProfileUpdatePage.skipSetUpLinkExists()).notOk();
-  await t.expect(await enrollProfileUpdatePage.getFormFieldLabel('userProfile.secondEmail')).eql('Secondary email');
-  await t.expect(enrollProfileUpdatePage.getFormFieldSubLabel('userProfile.secondEmail')).eql('Optional');
+  await t.expect(await enrollProfileUpdatePage.skipProfileLinkExists()).eql(false);
+  await t.expect(await enrollProfileUpdatePage.formFieldExistsByLabel('Secondary email')).eql(true);
+  // TODO: OKTA-524769 - awaitng ODY team to create wrapped components to enable optional sub label in v3
```

Comment:
👍 ok this is really the only time we can do something like this since we're planning to solve ASAP after a dependency is fulfilled.